invenio_communities/assets/semantic-ui/js/invenio_communities/subcommunity/SubcommunityResults.js

```diff
@@ -0,0 +1,75 @@
+// This file is part of Invenio
```

Comment:
This file is almost the same as https://github.com/inveniosoftware/invenio-requests/blob/master/invenio_requests/assets/semantic-ui/js/invenio_requests/search/RequestsResults.js (just renamed it). That makes me think this can be moved to a higher module so both requests and communities can reuse it.

Reply:
Maybe also in `react-invenio-forms`?

Reply:
this is a search aware component, we should move it to invenio-search-ui?

Reply:
> this is a search aware component, we should move it to invenio-search-ui?

Exactly, I was just messaging Pablo about this. +1 on moving it to `invenio-search-ui`

Reply:
After the discussion, I moved it to a reusable component named `invenio_search_ui/components/SearchResultsBox`. Not the best name but I'm open to changing it to anything else 😄 

However, let's align IRL whether this is the right place / way or not . Ping @ntarocco @ptamarit @kpsherva

Reply:
The idea was to create a base package, similar to invenio-base for Python code, that serves as a catalog of components. This package should remain stateless, without any specific features or logic, making it a simple dependency that can be included in other modules.

While I understand the rationale behind associating a component that displays search results with invenio-search-ui, I believe that it doesn't align with the intended purpose of the base package.

The base package is meant to be a neutral catalog, and incorporating such a component into invenio-search-ui would go against these criteria.

I am sure that we will find a case where we will need that component in a place where we don't depend on invenio-search-ui.

Happy to hear/read counterarguments.